src/utils/types.ts

```diff
@@ -63,10 +61,15 @@ export interface Address {
   street: string;
   city: string;
   state: string;
-  zipCode: string;
+  zipCode: number;
   country: string;
 }
 
+export interface MaintenancePeriod {
```

Comment:
This can be removed since we no longer care about the unit on the maintenance period.